### PR TITLE
Thomasg/load map id post changes: added cors, added size to nodes/relationships, etc

### PR DESCRIPTION
**Pull request by trg5pdx**
cors was added to prevent the front end from being blocked from sending requests
size was added in the create request for nodes and relationships
switched over to using post for get requests for nodes and edges

### CODE REVIEW

**Review by jgmccarthy7 — APPROVED** [review]
LGTM, TYVM

File: routes/node.js [hunk 1]
```diff
@@ -41,123 +49,117 @@ router.get("/", (req, res) => {
...
 	});
 });
 
 /**
  * POST request which takes userID, mapID, name, and color, and creates a new node record.
  */
 router.post("/create", (req, res) => {
-	const {userID, mapID, nodeinfo} = req.body;
+	const { userID, mapID, nodeinfo } = req.body;
+	console.log(userID);
```
Comment (jac0bml) [inline routes/node.js:86]: Are these console logs supposed to be present in production?

**Review by jac0bml — APPROVED** [review]
Looks good!

File: routes/node.js (hunk 1 above)
Comment (trg5pdx) [reply routes/node.js]: Whoops, I missed removing those. I just made a commit deleting those from this branch. Thanks for catching that!

**Pull request merged**